README.md

```diff
@@ -156,6 +148,50 @@ Configure the form-submit functionality which enables you to submit a (login) fo
 | input_filter      | A list of input fields that should not be submitted with the form                                         | False    |         | string - list       |
 | submit_once       | Submit the form only once on startup instead of each scan interval                                        | False    | False   | boolean             |
 | resubmit_on_error | Resubmit the form after a scraping error is encountered                                                   | False    | True    | boolean             |
+| variables         | See [Form Variables](#Form-Variables)                                                                     | False    |         | list                |
+
+### Form Variables
+
+Configure the variables you want to be forwarded from scraping the [Form-submit](#form-submit) page to scraping the main page for sensor data. A common use case is to populate the `X-Login-Token` header which is the result of the login.
+
+| name | description                                   | required | default | type   |
+| ---- | --------------------------------------------- | -------- | ------- | ------ |
+| name | Name of the variable                          | True     |         | string |
+|      | Shared fields from the [Selector](#Selector). | True     |         |        |
+
+Example:
+
```

Comment:
I don't like to use a specific (commercial) site here as an example. Could you create an anonymous example. Like: resource: "https://somesiteyouwanttoscrape.com"